src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml

```diff
@@ -728,6 +782,43 @@ components:
         nInstr:
           type: integer
           format: int32
+    InitiativesWithInstrumentDTO:
+      type: object
+      required:
+        - idWallet
+        - maskedPan
+        - brand
+        - initiativeList
+      properties:
+        idWallet:
+          type: string
+        maskedPan:
+          type: string
+        brand:
+          type: string
+        initiativeList:
+          type: array
+          items:
+            $ref: '#/components/schemas/InitiativesStatusDTO'
+          description: The list of the payment instrument status with respect to the initiative
+    InitiativesStatusDTO:
+      type: object
+      required:
+        - initiativeId
+        - status
+        - instrumentId
+      properties:
+        initiativeId:
+          type: string
+        idInstrument:
+          type: string
+        status:
+          enum:
+            - ACTIVE
+            - INACTIVE
+            - PENDING_ENROLLMENT_REQUEST
+            - PENDING_DEACTIVATION_REQUEST
+            - ENROLLMENT_FAILED
```

Comment:
please add a `type` and `description` field to the enum, else the generator will fail on `ErrorDTO`

Reply:
@forrest57 fixed in [008170a](https://github.com/pagopa/cstar-infrastructure/pull/832/commits/008170a48e1c5e40156552d9c22fa44b1cc09def)